docs/site/guides/deployment/enabling-https.md

```diff
@@ -0,0 +1,62 @@
+---
+lang: en
+title: 'Enabling HTTPS'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, HTTPS, Deployment
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Enabling-https.html
+---
+
+Enabling HTTPS for the LoopBack REST server is just a matter of specifying the
+protocol as `https` and specifying the credentials.
```

Comment:
"...specifying the credentials." -> "...specifying the credentials in the rest options of the application configuration."?

Reply:
Sounds good, addressed in 9871197.